### ISSUE TITLE
A Daring Rescue - NPC stucks, when running up on stairs after agro is gone

### ISSUE THREAD

**Issue by Invizet** [echoes-emu/echoes-bt#761]
NPC stucks, when running up on stairs after agro is gone.

Selection: 0x031876F0000005AF (0x041876F00000000F)
 DID: 0x70000C86
 Loc: bx135 by110 c0 ox62.5 oy31.9013 oz455.07 r128 (Instance 0)
 Move table: 0x070001BD
State stack:
(Patrol) patrol_obj1_start @ 0.7x
(Interacting)
(Interacting)
(Idle)
Melee combat mode: combat_monster
Ranged combat mode: combat_monster
KungFu NPC: No

https://youtu.be/-m-u4hAp1Qo (sorry for 'white noise').

**Comment by woodx319** [comment]
is this in an instance or open world @Invizet ?

**Comment by bcp19** [comment]
Open world

**Comment by echoes-emu** [comment]
Merging with #907

**Comment by woodx319** [comment]
Just adding some corroborating info 

Ran this two times. 

The first time I was able to beat the quest but Leofwenna jumped boldly off a bridge pass at one point. 

Second time I ran it she jumped off that same bridge but then got stuck on a different ledge either kneeling or anti exploit mode if I pulled some mobs to her to try to get her motivated again. 


**Quest Original Location:** 
!goto bx135 by111 c0 ox107.239 oy89.9785 oz473.441 r151 (Instance 0)

**Bridge derp location**
You are at: bx135 by110 c0 ox19.1923 oy26.4635 oz467.693 r70 (Instance 0)

**Stuck Location**
You are at: bx135 by110 c0 ox48.8635 oy8.1538 oz460.515 r130 (Instance 0)


![ScreenShot00027](https://user-images.githubusercontent.com/104697795/224820859-6140c823-86dc-4755-925f-285ba986055f.jpg)
![ScreenShot00028](https://user-images.githubusercontent.com/104697795/224820893-785c9963-b5ea-4f7b-afb9-7ecbcec2157a.jpg)
